include/networkit/io/NetworkitBinaryGraph.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef NETWORKIT_BINARY_GRAPH_
+#define NETWORKIT_BINARY_GRAPH_
+
+struct Header {
+	char magic[8];
```

Comment:
Regarding @michitux's comment about endianness: Would it make sense to declare the magic word as uint64_t and specify that the writer uses its native endianness? Then, a later revision of the reader could support different coding standards. This would achieve maximal performance if the file remains on one system. (also the majority of data will likely be stored in varints where endianess is well-defined)

Reply:
Regarding the endianness: The vast majority of all systems are probably little endian today (AFAIK this applies even to ARM, which supports both in theory). Hence, I would prefer to specify the format as little endian. Big endian machines can probably convert the few fixed-size numbers without huge performance issues.